lib/board.rb

```diff
@@ -42,15 +43,16 @@ def view_board
     @inOut.print(@board)
   end
   
-  def is_square_free?(square)
-    puts @squares[square].is_square_free?
-    @squares[square].is_square_free?
+  def is_square_free?(square_number)
+    if @validate.is_valid_player_input?(square_number, @max_turns)
```

Comment:
Should board be validating the input? Or could the input be validated before it gets to board?